.github/workflows/generate-event-pdfs.yml

```diff
@@ -2,7 +2,18 @@ name: generate-pdf-artifacts
 
 on:
   workflow_dispatch:
-    inputs: {}
+    inputs:
+      REMOVE_MASTER:
+        description: Remove the master files
+        required: false
+        default: '1'
+  pull_request:
+    paths:
+      - '_events/*/*.md'
+
+env:
+  REMOVE_MASTER: 1
```

Comment:
I'm wondering why you have both the input variable and the environment variable? Can you walk me through how you see a user changing either of those, and why we can't just use the single input variable?